source/_integrations/scrape.markdown

```diff
@@ -76,6 +76,10 @@ headers:
   description: Headers to use for the web request.
   required: false
   type: string
+scan_interval:
```

Comment:
This is not part of this integration, but part of the platform options of Home Assistant:

<https://www.home-assistant.io/docs/configuration/platform_options/#scan-interval>

And this applies to all platforms.